packages/apps/transfer/src/components/Common/Layout/partials/Footer/Footer.tsx

```diff
@@ -0,0 +1,100 @@
+import { Footer, SystemIcon, useModal } from '@kadena/react-ui';
+
+import { linkClass } from './styles.css';
+
+import { OptionsModal } from '@/components/Global/OptionsModal';
+import Link from 'next/link';
+import { useTheme } from 'next-themes';
+import useTranslation from 'next-translate/useTranslation';
+import React, { FC } from 'react';
+
+const FooterWrapper: FC = () => {
+  const { theme, setTheme } = useTheme();
```

Comment:
Is the theme preference saved in the session? For example if it is dark and you set it to light and reload the page, will it still be light?

Reply:
Yes, it's saved in storage, so on reload you'll get the last set theme